engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql

```diff
@@ -641,3 +641,8 @@ CREATE VIEW `cloud`.`domain_router_view` AS
         `cloud`.`async_job` ON async_job.instance_id = vm_instance.id
             and async_job.instance_type = 'DomainRouter'
             and async_job.job_status = 0;
+
+INSERT INTO `cloud`.`user_vm_details`(`vm_id`, `name`, `value`)
```

Comment:
@davidjumani 
is it possible that ssh keypairs are under different account but have same public key ?
for example, a user register a public key under ROOT/admin, then register same public key in his owner account ?

Reply:
It is possible but an edge case since users would not register the same key twice. I don' think we need to consider this scenario

Reply:
@davidjumani 
I think it would be better to consider all possible scenarios.

my sugguestion is

```
INSERT INTO `cloud`.`user_vm_details`(`vm_id`, `name`, `value`)
SELECT `user_vm_details`.`vm_id`, 'SSH.KeyPairNames', `ssh_keypairs`.`keypair_name`
        FROM `cloud`.`user_vm_details`
        INNER JOIN `cloud`.`ssh_keypairs` ON ssh_keypairs.public_key = user_vm_details.value
        INNER JOIN `cloud`.`vm_instance` ON vm_instance.id = user_vm_details.vm_id
        WHERE ssh_keypairs.account_id = vm_instance.account_id
```

Reply:
@davidjumani @nvazquez 
what's your opinion on this comment ?

Reply:
@weizhouapache Makes sense. I'll make the changes

Reply:
Was this addressed? This is NOT an edge case, I have customers using this setup extensively.

There is also a unique key constraint in the table, so pls make sure this is not a problem

Reply:
Yes it's been addressed